src/vm/gccover.cpp

```diff
@@ -1371,72 +1479,69 @@ void DoGcStress (PCONTEXT regs, MethodDesc *pMD)
 
     Thread *pThread = GetThread();
 
+    if (!IsGcCoveregeInterruptInstruction((LPVOID)instrPtr))
+    {
+        _ASSERTE(IsOriginalInstruction((LPVOID)instrPtr, gcCover, offset));
+        // Someone beat us to it, just go on running.
+        return;
+    }
+
+    bool atCall;
+    bool afterCallProtect[2] = { 0 };
```

Comment:
Make it more explicit to be clear? `= { false, false }`?

Reply:
Done.